rotkehlchen/premium/sync.py

```diff
@@ -247,7 +248,7 @@ def maybe_upload_data_to_server(
         self.last_upload_attempt_ts = self.last_data_upload_ts
         self.last_remote_data_upload_ts = self.last_data_upload_ts
         with self.data.db.user_write() as cursor:
-            self.data.db.set_setting(cursor, name='last_data_upload_ts', value=self.last_data_upload_ts)  # noqa: E501
+            self.data.db.set_cache(cursor, DBCache(last_data_upload_ts=self.last_data_upload_ts))
```

Comment:
if you do this aren't you overwritting all the other values in the database with the default? `0`

Reply:
This is why I like the other interface more. Needing to build all the object to modify a single entry is cumbersome